mm2src/lp_ordermatch.rs

```diff
@@ -4305,6 +4348,13 @@ pub async fn create_maker_order(ctx: &MmArc, req: SetPriceReq) -> Result<MakerOr
     };
 
     let ordermatch_ctx = try_s!(OrdermatchContext::from_ctx(ctx));
+    let ticker = base_coin.ticker().to_string();
```

Comment:
to_string is required only if !coin_has_order, please do it only when this condition is met.